posts/2023-08-22-23.0.0.8.adoc

```diff
@@ -0,0 +1,198 @@
+author_github: https://github.com/mbroz2
+seo-title: Prevent authorization code interception attacks with PKCE support for OpenID Connect clients - OpenLiberty.io
+seo-description: PKCE support in OpenID Connect clients to prevent authorization code interception attacks in certain specific contexts. Also, changes to the featureUtility installFeature command, a new OpenTelemetry and Jaeger guide, and bug fixes.
+blog_description: PKCE support in OpenID Connect clients to prevent authorization code interception attacks in certain specific contexts. Also, changes to the featureUtility installFeature command, a new OpenTelemetry and Jaeger guide, and bug fixes.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Prevent authorization code interception attacks with PKCE support for OpenID Connect clients in Open Liberty 23.0.0.8
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+With Open Liberty's new Proof Key for Code Exchange (PKCE) support in OpenID Connect clients, you can prevent authorization code interception attacks, which can occur in certain very specific scenarios. Also in this release, the `featureUtility installFeature` command is updated to better manage dependencies among the features that it installs. We've also got a new guide on using OpenTelemetry and Jaeger and, of course, we've been fixing bugs. 
+
+
+In link:{url-about}[Open Liberty] 23.0.0.8:
+
+* <<pkce, Prevent authorization code interception attacks with PKCE support for OpenID Connect clients>>
+* <<featureinstall, Ensure sufficient features are installed when using `featureUtility installFeature` command>>
+* <<CVEs, Security Vulnerability (CVE) fixes>>
+
+Along with the new features and functions added to the runtime, we’ve also made <<guides, updates to our guides>>.
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23008+label%3A%22release+bug%22[23.0.0.8].
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
+
+
+[#run]
+
+== Run your apps using 23.0.0.8
+
+For link:{url-prefix}/guides/maven-intro.html[Maven projects], use the link:https://github.com/OpenLiberty/ci.maven/blob/main/README.md[Liberty Maven Plugin] for quicker and easier development:
+
+[source,xml]
+----
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>3.8.2</version>
+    <configuration>
+      <runtimeArtifact>
+         <version>23.0.0.8</version>
+      </runtimeArtifact>
+    </configuration>
+</plugin>
+----
+
+For link:{url-prefix}/guides/gradle-intro.html[Gradle projects], use the link:https://github.com/OpenLiberty/ci.gradle/blob/main/README.md[Liberty Gradle Plugin] for quicker and easier development:
```

Comment:
Can we point to a section within the README? For gradle add `#usage` and for maven add `#plugins` to the link.

Reply:
Yes, done, thanks